content/en/docs/concepts/sdk-configuration/sdk-environment-variables.md

```diff
@@ -0,0 +1,164 @@
+# OpenTelemetry Environment Variable Specification
+
+**Status**: [Mixed](document-status.md)
+
+The goal of this specification is to unify the environment variable names between different OpenTelemetry SDK implementations. SDKs MAY choose to allow configuration via the environment variables in this specification, but are not required to. If they do, they MUST use the names listed in this document.
```

Comment:
This is more-or-less just copying text from the specification. I don't think we should do that. The audience for the specification is not end users, it is implementers of opentelemetry SDKs. For end users we need to phrase things differently, or in some cases, simply elide some text because it's just not relevant for anyone other than implementers of opentelemetry SDKs.

Reply:
I agree with that point. I plan to go through each of the environmental variables and add specific and relevant information to them.

Reply:
Thanks @siddhant-khisty (and @cartermp)!